cmake/dependencies.cmake

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Leidos
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include(GetCPM)
+set(CPM_USE_LOCAL_PACKAGES ON)
+
+# lint_cmake: -readability/wonkycase
+CPMAddPackage(NAME units
+  GITHUB_REPOSITORY nholthaus/units
+  GIT_TAG v2.3.3
+  SYSTEM ON
+  OPTIONS
+    "BUILD_TESTS FALSE"
+    "BUILD_DOCS FALSE"
+)
+#TODO: This is probably not a complete working fix but it allows that project install
+# target to run successfully. Needs tot be updated
+export( TARGETS units NAMESPACE units:: FILE unitsConfig.cmake)
```

Comment:
This is unnecessary because the `units` library always creates install targets.

Reply:
When I remove this I get the following CMake error:
```
Make Error in CMakeLists.txt:
  export called with target "carma-geodetic" which requires target "units"
  that is not in any export set.
```

Reply:
This is a problem with the carma-builds `InstallingConfig.cmake` file, which makes a call to `export(TARGETS)`. I believe this requires all targets in the export set to also have their import targets available immediately. I could be wrong here though. In any case, I don't think the `export(TARGETS)` call is need here nor in `InstallingConfig.cmake`. We should open a new issues in the carma-builds repo about this.